api/aperture/flowcontrol/v1/flowcontrol.proto

```diff
@@ -83,3 +85,9 @@ message LimiterDecision {
 message FluxMeter {
   string flux_meter_name = 1;
 }
+
+// Classifier describes details for each Classifier.
+message Classifier {
+  string policy_name = 1;
+  string policy_hash = 2;
```

Comment:
Need `classifier_index` field since there could be more than one classifier per policy